native/src/components/FeedbackContainer.tsx

```diff
@@ -70,20 +88,33 @@ const FeedbackContainer = ({ query, language, routeType, cityCode, slug }: Feedb
     })
   }
 
+  const fallbackLanguage = config.sourceLanguage
+
   return (
     <Container>
-      <Feedback
-        comment={comment}
-        contactMail={contactMail}
-        sendingStatus={sendingStatus}
-        onCommentChanged={setComment}
-        onFeedbackContactMailChanged={setContactMail}
-        isPositiveFeedback={isPositiveRating}
-        setIsPositiveFeedback={setIsPositiveRating}
-        onSubmit={handleSubmit}
-        searchTerm={searchTerm}
-        setSearchTerm={setSearchTerm}
-      />
+      {showFeedback ? (
+        <Feedback
+          comment={comment}
+          contactMail={contactMail}
+          sendingStatus={sendingStatus}
+          onCommentChanged={setComment}
+          onFeedbackContactMailChanged={setContactMail}
+          isPositiveFeedback={isPositiveRating}
+          setIsPositiveFeedback={setIsPositiveRating}
+          onSubmit={handleSubmit}
+          searchTerm={searchTerm}
+          setSearchTerm={setSearchTerm}
+        />
+      ) : (
+        <>
+          <BoldText>
+            {language === fallbackLanguage ? t('noResultsInOneLanguage') : t('noResultsInTwoLanguages')}
+          </BoldText>
+          <Text>{t('checkQuery', { appName: buildConfig().appName })}</Text>
+          <TextBeforeButton>{t('informationMissing')}</TextBeforeButton>
+          <TextButton text={t('giveFeedback')} onPress={() => setShowFeedback(true)} />
+        </>
+      )}
     </Container>
   )
```

Comment:
:upside_down_face: Very personal opinion, so feel free to ignore. I think if there is only this little things to reuse, its a lot easier to read to add a separate if-clause instead of an JSX inline if:
``` tsx
if (showFeedback) {
  return <Container><Feedback.../></Container>
}
return <Container><Hint.../></Container>
```